account_journal_general_sequence/models/account_journal.py

```diff
@@ -13,27 +13,20 @@ class AccountJournal(models.Model):
     entry_number_sequence_id = fields.Many2one(
         comodel_name="ir.sequence",
         string="Account entry number sequence",
-        default=lambda self: self._default_entry_number_sequence(),
+        compute="_compute_entry_number_sequence",
         copy=False,
```

Comment:
Maybe add `store=True`, `readonly=False` to allow select other sequence? Also add domain for free chosing sequences on a secure multi-company environment.
```suggestion
        copy=False,
        store=True,
        readonly=False,
        domain="[('company_id', '=', company_id)]",
```